api/src/threads/dto/create-thread.dto.ts

```diff
@@ -1,5 +1,9 @@
 import { IntersectionType, PickType } from '@nestjs/mapped-types';
-import { IsEnum, IsString, Length } from 'class-validator';
+import {
+  IsEnum,
+  IsString,
+  Length,
+} from 'class-validator';
```

Comment:
```suggestion
import { IsEnum, IsString, Length } from 'class-validator';
```
*idem, je suis manique*